src/Dhadgar.Scope/src/components/layout/MobileDrawer.tsx

```
@@ -1,76 +1,76 @@
+        className="fixed inset-y-0 left-0 z-[1001] w-[min(92vw,380px)] overflow-y-auto border-r border-white/10 bg-slate-950/95 backdrop-blur-xl lg:hidden"
+        style={{ transform: isOpen ? "translateX(0)" : "translateX(-100%)" }}
+      >
+        {/* Header */}
+        <div className="flex items-center justify-between border-b border-white/10 p-4">
+          <span className="text-sm text-white/60">Menu</span>
+          <button
+            type="button"
+            onClick={onClose}
+            className="rounded-lg border border-white/10 bg-white/5 p-1.5 transition-colors hover:bg-white/10"
+          >
+            <svg className="h-4 w-4" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)